evm/estimators/__init__.py

```diff
@@ -0,0 +1,14 @@
+import os
+
+from evm.utils.module_loading import (
+    import_string,
+)
+
+
+def get_gas_estimator(import_path=None):
+    if not import_path:
+        import_path = os.environ.get(
+            'GAS_ESTIMATOR_BACKEND_FUNC',
+            'evm.estimators.gas.binary_search_intrinsic_tolerance',
+        )
+    return import_string(import_path)
```

Comment:
Note to self:  this pattern is repeated at least one other place.  We should write a util in `evm.utils.env` to do the combination of pulling a value from the environment and then importing it.

Reply:
I thought about this a little more, and I can't come up with a time when we would want to pass in an `import_path` here (or in `get_db_backend()`). If we want to specify either one explicitly, we can just import and use it directly. `get_db_backend()` is never called with an argument in the code base, although it supports it. I think supporting an `import_path` here is a YAGNI feature, which means this just simplifies down to:

```py
def get_gas_estimator():
    import_path = os.environ.get(
        'GAS_ESTIMATOR_BACKEND_FUNC',
        'evm.estimators.gas.binary_search_intrinsic_tolerance',
    )
    return import_string(import_path)
```

I'm not sure there's much left there to generalize.